code/game/objects/items/manuals.dm

```diff
@@ -248,13 +247,13 @@
 
 /obj/item/book/manual/wiki/proc/initialize_wikibook()
 	//var/wikiurl = CONFIG_GET(string/wikiurl) //ORIGINAL
-	//SKYRAT EDIT CHANGE BEGIN
+	//SKYRAT EDIT ADDITION START - Wiki overrides
 	var/wikiurl
 	if(skyrat_wiki)
 		wikiurl = CONFIG_GET(string/wikiurlskyrat)
 	else
 		wikiurl = CONFIG_GET(string/wikiurl)
-	//SKYRAT EDIT CHANGE END
+	//SKYRAT EDIT ADDITION END - Wiki overrides
```

Comment:
So, I did some more digging on the server, and turns out that this is entirely unnecessary, because we have nearly all of the pages already on our wiki, with edits to account for Skyrat-exclusive content as well.

The proper fix here would just be to remove the whole `wikiurlskyrat` config, as it seems it was never used on the server either way.

I verified, and turning the `skyrat_wiki` variable to `0` on the Corporate Regulations book fixes it, and makes it display what it's meant to display.

To make it work on test-servers too, you could instead change the default value of the `wikiurl` entry, with a modular comment (or completely modularly, up to you).

Reply:
God this has been an adventure

If I override it to default to the skyrat wiki, I have to add 'wiki.' to the beginning lmao
This suckssss

I'll have it done sooner or later